programmers/크레인-인형뽑기-게임.js

```diff
@@ -0,0 +1,37 @@
+function pullDoll(board, move) {
+  const dollIndex = board.findIndex((column) => column[move] !== 0);
+  if (dollIndex === -1) return -1;
+  const doll = board[dollIndex][move];
+  board[dollIndex][move] = 0;
+  return doll;
+}
```

Comment:
줄바꿈 신경써주세요.
로직과 변수 선언 부분은 구분해주는 것이 좋습니다.
```suggestion
function pullDoll(board, move) {
  const dollIndex = board.findIndex((col) => col[move] !== 0);
  if (dollIndex === -1) return -1;
  
  const doll = board[dollIndex][move];
  board[dollIndex][move] = 0;
  return doll;
}
```